Makefile

```diff
@@ -56,6 +56,7 @@ open-coverage:
 
 # Run complete tox test suite in a multi-python Docker container
 .PHONY: docker-tox
+docker-tox: TOX_ARGS='-e clean,py312,py311,py310,py39,report,flake8,py312-mypy'
```

Comment:
Two docker-tox entries? Can this be correct?

Reply:
I guess it is valid, and that is just means that now we have two "instructions".

Reply:
The first one just sets the default for the TOX_ARGS variable (which is overridden in the same way by the `docker-test-py*` targets), the second one defines the commands for it.

It's just a Makefile weirdness :sweat_smile: